docs/benchmark.md

```diff
@@ -0,0 +1,139 @@
+# Benchmarking SOCI snapshotter
+
+This document walks through how to run the SOCI snapshotter benchmarks, including running the benchmarks on default and custom workloads.
+
+- [Prerequisites](#prerequisites)
+- [Running benchmarks on default workloads](#running-benchmarks-on-default-workloads)
+- [Running benchmarks on custom workloads](#running-benchmarks-on-custom-workloads)
+- [Benchmark binaries cli flags](#benchmark-binaries-have-different-cli-flags)
+- [Csv file format for custom workloads](#csv-file-format-for-custom-workloads)
+- [Default custom workloads](#default-custom-workloads)
+- [Benchmark results format](#benchmark-results)
+
+## Prerequisites
+
+Follow the [Getting started guide](/docs/getting-started.md) and complete setting up the project.
+
+
+## Running benchmarks on default workloads
+
+There a set of [8 workloads](#default-custom-workloads) that are included in the benchmark binaries that can readily be benchmarked without any additional setup.
+These workloads are hosted on a public ECR repository and hence do not need any credetials.
+
+```make benchmarks```  - Runs the benchmark tests on the workloads 5 times
+
+```make benchmarks-perf-test``` - Runs only the performance benchmark tests on the workloads 5 times
```

Comment:
It's not clear without prior knowledge what "performance benchmark" tests are and how they differ from the other benchmarks. I think it would be useful here to define that `make benchmarks` will compare SOCI to overlayfs and `make benchmarks-perf-test` will only run SOCI benchmarks.